### PR TITLE
Added EZChain mainnet, testnet and icon

### PR DESCRIPTION
**Pull request by enixdark**
Add EZChain (EZC) Chain ID: 2612 for mainet and 2613 for testnet

### CODE REVIEW

**Comment by enixdark** [comment]
hi, can you can review this PR!

File: _data/chains/eip155-2612.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  },
+  "infoURL": "https://ezchain.com/",
+  "shortName": "EZChain",
+  "chainId": 2612,
+  "networkId": 2612,
+  "icon": "ezchain",
+  "explorers": [
+    {
+      "name": "ezchain",
+      "url": "https://cchain-explorer.ezchain.com/",
```
Comment (ligi) [inline _data/chains/eip155-2612.json:20]:
```suggestion
      "url": "https://cchain-explorer.ezchain.com/",
```

File: _data/chains/eip155-2613.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "infoURL": "https://ezchain.com/",
+  "shortName": "EZChain",
+  "chainId": 2613,
+  "networkId": 2613,
+  "icon": "ezchain",
+  "explorers": [
+    {
+      "name": "ezchain",
+      "url": "https://testnet-cchain-explorer.ezchain.com/",
```
Comment (ligi) [inline _data/chains/eip155-2613.json:23]:
```suggestion
      "url": "https://testnet-cchain-explorer.ezchain.com",
```

**Review by ligi — CHANGES_REQUESTED** [review]
Please remove the trailing slashes as suggested

File: _data/chains/eip155-2613.json (hunk 2 above)
Comment (enixdark) [reply _data/chains/eip155-2613.json]: i have just removed trailing slash

**Comment by enixdark** [comment]
i have fixed faucets in mainnet after building fail.

**Pull request merged**